tests/test-all-mcus.stanza

```diff
@@ -0,0 +1,23 @@
730ZBT6" "STM32H733VGT6" "STM32H733ZGT6" "STM32H735AGI6" "STM32H735IGT6" "STM32H735ZGT6" "STM32H745BGT6" "STM32H745BIT6" "STM32H745BIT7" "STM32H745IGT6" "STM32H745IIT6" "STM32H745IIT7" "STM32H745ZGT6" "STM32H745ZIT6" "STM32H745ZIT7" "STM32H747AGI6" "STM32H747AII6" "STM32H747BGT6" "STM32H747BIT6" "STM32H747IGT6" "STM32H747IIT6" "STM32H747ZIY6" "STM32H750IBT6" "STM32H750VBT6" "STM32H750ZBT6" "STM32H755BIT6" "STM32H755BIT7" "STM32H755IIT6" "STM32H755IIT7" "STM32H755ZIT6" "STM32H755ZIT7" "STM32H757AII6" "STM32H757BIT6" "STM32H757IIT6" "STM32H757ZIY6" "STM32H7A3AGI6Q" "STM32H7A3AII6Q" "STM32H7A3IGT6Q" "STM32H7A3IIT6Q" "STM32H7A3LGH6Q" "STM32H7A3LIH6Q" "STM32H7A3NGH6" "STM32H7A3NIH6" "STM32H7A3VGT6Q" "STM32H7A3VIT6Q" "STM32H7A3ZGT6Q" "STM32H7A3ZIT6Q" "STM32H7B0ABI6Q" "STM32H7B0IBT6" "STM32H7B0VBT6" "STM32H7B0ZBT6" "STM32L010C6T6" "STM32L031C4U6" "STM32L031C6U6" "STM32L031E4Y6" "STM32L031E6Y6" "STM32L031G4U6" "STM32L031G6U6" "STM32L031G6U6S" "STM32L031G6U7S" "STM32L051C6U6" "STM32L051C8U6" "STM32L051T6Y6" "STM32L051T8Y6" "STM32L052C6U6" "STM32L052C8U6" "STM32L052T6Y6" "STM32L052T6Y7" "STM32L052T8F6" "STM32L052T8F7" "STM32L052T8Y6" "STM32L052T8Y7" "STM32L053C6U6" "STM32L053C6U7" "STM32L053C8U6" "STM32L053C8U7" "STM32L062C8U6" "STM32L063C8U6" "STM32L071C8U6" "STM32L071C8U7" "STM32L071CBY6" "STM32L071CBY7" "STM32L071CZY6" "STM32L071CZY7" "STM32L071K8U6" "STM32L071K8U7" "STM32L071KBU6" "STM32L071KBU7" "STM32L071KZU6" "STM32L071KZU7" "STM32L071V8T6" "STM32L071VBT6" "STM32L071VBT7" "STM32L071VZT6" "STM32L071VZT7" "STM32L073CBU6" "STM32L073CZU6" "STM32L073CZY6" "STM32L073CZY7" "STM32L073RZI6" "STM32L073RZI7" "STM32L073V8T6" "STM32L073V8T7" "STM32L073VBT6" "STM32L073VBT7" "STM32L073VZT6" "STM32L073VZT7" "STM32L081CBT6" "STM32L081CZT6" "STM32L081CZU6" "STM32L081KZU6" "STM32L082CZY6" "STM32L082CZY7" "STM32L082KBU6" "STM32L082KZU6" "STM32L083CBT6" "STM32L083CZT6" "STM32L083CZU6" "STM32L083V8T6" "STM32L083VBT6" "STM32L083VZT6" "STM32L100C6U6" "STM32L100C6U6A" "STM32L151C6U6A" "STM32L151C8U6A" "STM32L151CBU6A" "STM32L151CCU6" "STM32L151CCU7" "STM32L151QDH6" "STM32L151QEH6" "STM32L151RCY6" "STM32L151RDY6" "STM32L151RDY7" "STM32L151UCY6" "STM32L151UCY7" "STM32L151V8T6A" "STM32L151VBT6A" "STM32L151VCT6" "STM32L151VDT6" "STM32L151VDY6X" "STM32L151VDY7X" "STM32L151VEY6" "STM32L151VEY7" "STM32L151ZDT6" "STM32L151ZET6" "STM32L152C6U6A" "STM32L152C8U6A" "STM32L152CBU6A" "STM32L152CCU6" "STM32L152QDH6" "STM32L152QEH6" "STM32L152RDY6" "STM32L152UCY6" "STM32L152V8T6A" "STM32L152VBT6A" "STM32L152VCT6" "STM32L152VDT6" "STM32L152VDT6X" "STM32L152VEY6" "STM32L152ZDT6" "STM32L152ZET6" "STM32L162QCH6" "STM32L162QDH6" "STM32L162RDY6" "STM32L162VCT6" "STM32L162VCT6A" "STM32L162VDT6" "STM32L162VDY6X" "STM32L162VEY6" "STM32L162ZCT6" "STM32L162ZDT6" "STM32L162ZET6" "STM32L475VCT6" "STM32L475VET6" "STM32L475VGT6" "STM32L476MEY6" "STM32L476MGY6" "STM32L476QEI6" "STM32L476QGI6" "STM32L476VCT6" "STM32L476VET6" "STM32L476VGT6" "STM32L476ZET6" "STM32L476ZGT6" "STM32L476ZGT6P" "STM32L476ZGT7P"]
```

Comment:
I would recommend to move them in a data file (`txt` file?), with one package per line. Then, here, read the file.